Source/DataSources/Entity.js

```diff
@@ -342,6 +342,16 @@ Object.defineProperties(Entity.prototype, {
       this._definitionChanged.raiseEvent(this, "parent", value, oldValue);
     },
   },
+  /**
+   * Gets the list of all children currently associated with this entity
+   * @memberof Entity.prototype
+   * @type {Entity[]}
```

Comment:
> So is there any point in adding the @readonly argument to the jsdoc block or should there just be a different implementation of this?

Not if this specific getter goes away, I just wanted to quickly point it out in case it comes up again.

> We could also spread or copy the original array to prevent it from being mutated as well.

I see what you're saying, but I'd hesitate to recommend this way. I don't see anything similar in the rest of the codebase, most likely for performance reasons. It could get expensive if the getter is called frequently and if there are enough entities.